config/locales/en.yml

```diff
@@ -965,6 +965,15 @@ en:
         sample_does_not_belong_to_project: "Sample '%{sample_name}'' does not belong to project '${project_name}'"
         empty_metadata: "No metadata was received to update sample '%{sample_name}'"
         user_cannot_update_metadata: "Metadata fields '%{metadata_fields}' of sample '%{sample_name}' cannot be overwritten by a user as it was previously updated with an analysis. "
+        import_file:
+          empty_file: The file cannot be empty.
+          empty_sample_id_column: The sample id column cannot be empty.
+          invalid_file_extension: The file can only be a csv or excel.
+          duplicate_column_names: The file has duplicate column header names.
```

Comment:
Can we update this to also advise the user that they need to remove the duplicate column headers

Reply:
bcffae4ee9a78635022774876ba167180332d528

Reply:
9ffc2302f851ae1b9488a395421234d63e38ef98